starlette/middleware/sessions.py

```diff
@@ -64,7 +64,7 @@ async def send_wrapper(message: Message) -> None:
                 elif not initial_session_was_empty:
                     # The session has been cleared.
                     headers = MutableHeaders(scope=message)
-                    header_value = "%s=%s; %s" % (
+                    header_value = "{}={}; {}".format(
```

Comment:
Please can we keep small changes like this and the others (starlette/routing.py, tests/test_staticfiles.py) out of this PR if they're unrelated to the purpose of improving pytest stuff? We try to keep PRs small and very focussed.

Reply:
This was originally to fix the SyntaxWarning with `pyupgrade`